### DISCUSSION TITLE
客户端获取ipv6后只有Netflix和Disney+上不去，求解惑

### DISCUSSION THREAD

**Discussion by gzm930508** [v2fly/v2ray-core#1565] [Q&A]
华硕路由器 梅林固件 软件中心科学上网插件
如果打开ipv6，正常google、yt等都可以访问，但是Netflix和Disney+不行。
Netflix页面显示“Not Available”
Disney+页面显示“Sorry, Disney+ is not available in your region.”
看了服务端的access.log日志，没有这两个网站的访问记录。

但是把ipv6关掉，客户端只有ipv4地址时，打开这两个网站正常，也能播放视频，access.log日志里也有访问记录。

求解决方法，谢谢

**Comment by ghost** [comment]
IPv6 transparent proxy not available on router right now.
Actually, not any resolve ways for this time.
